xla/service/gpu/fusions/cudnn.h

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2024 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef XLA_SERVICE_GPU_CUDNN_FUSION_H_
+#define XLA_SERVICE_GPU_CUDNN_FUSION_H_
```

Comment:
Linter complains, should be XLA_SERVICE_GPU_FUSIONS_CUDNN_H_

Reply:
This one is not fixed yet

Reply:
Fixed.